assets/src/blocks/ENForm/ENFormFrontend.js

```diff
@@ -159,7 +159,7 @@ const Signup = ({attributes, fields, form_data, onInputChange, onBlur, onFormSub
     <div className="enform" id="enform">
       <div id="enform-content">
 
-        <div className="title-and-description">
+        <h4 className="title-and-description">
```

Comment:
This should be semantically a heading, then we should avoid to create extra classes to style the title.